15.2.0-rc5: ccm-fetch broken

This is a showstopper for 15.2.0-rc5 (https://github.com/quattor/release/issues/81). After ugrading to new `ccm` version, any attempt to run `ncm-ccm` fails with the error below but `ccm-fetch` ran outside the component works fine. As there was almost no change to `ncm-ccm`since last release (only `NOQUATTOR` support), I tend to think the problem is in `CCM`

```
[INFO] running component: ccm
---------------------------------------------------------
[ERROR] failed to ccm-fetch with new config: Uncaught exception!!! Calling stack is:
  LC::Exception::throw_error called at /usr/lib/perl/EDG/WP4/CCM/CCfg.pm line 193
  EDG::WP4::CCM::CCfg::initCfg called at /usr/lib/perl/EDG/WP4/CCM/Fetch.pm line 161
  EDG::WP4::CCM::Fetch::_config called at /usr/lib/perl/EDG/WP4/CCM/Fetch.pm line 140
  EDG::WP4::CCM::Fetch::new called at /usr/sbin/ccm-fetch line 222
*** _readConfigFile(/proc/self/fd/0): non-existing config file /proc/self/fd/0

[INFO] configure on component ccm executed, 1 errors, 0 warnings
```

--- a/src/main/perl/CCfg.pm
+++ b/src/main/perl/CCfg.pm
@@ -104,10 +104,6 @@ sub _readConfigFile ($)
 {
     my ($fn) = @_;
 
-    if (! -f $fn) {
-        throw_error("non-existing config file $fn");
-        return;        
-    }
     my $fh = CAF::FileReader->new($fn);
         
     foreach my $line (split ("\n", "$fh")) {
@@ -160,7 +156,7 @@ sub initCfg
 
         # Accept the configuration be read from pipes (i.e, stdin)
         unless (-f $cp || -p $cp) {
-            throw_error("configuration file not found");
+            throw_error("configuration file $cp not found");
             return ();
         }
     } else {

--- a/src/test/perl/test-ccfg.t
+++ b/src/test/perl/test-ccfg.t
@@ -50,7 +50,7 @@ my $fn = "$ccfgtmp/notexists";
 my $fh = CAF::FileReader->new($fn);
 ok(!-f $fn, "notexists file $fn doesn't exist");
 
-eok($ec, EDG::WP4::CCM::CCfg::_readConfigFile($fn), "_readConfigFile of not existing file");
+eok($ec, EDG::WP4::CCM::CCfg::initCfg($fn), "initCfg of not existing file");
 
 $fn = "$ccfgtmp/ccm_invalidkey.cfg";
 ok(-f $fn, "ccm_invalidkey file $fn does exist");

